File: php/structure-in-rte/RegisterLinkResolver.php
<?php
// DocSection: structure_in_rte_register_link_resolver
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
$client = new DeliveryClient("<YOUR_ENVIRONMENT_ID>");
$client->contentLinkUrlResolver = new CustomContentLinkUrlResolver(); 
// EndDocSection
?>

File: ruby/structure-in-rte/RegisterLinkResolver.rb
# DocSection: structure_in_rte_register_link_resolver
# Tip: Find more about Ruby SDKs at https://docs.kontent.ai/ruby
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>',
               content_link_url_resolver: link_resolver
# EndDocSection